docs/sources/developers/plugins/add-support-for-explore-queries.md

```diff
@@ -85,25 +85,20 @@ The query editor for Explore is similar to the query editor for the data source
    };
    ```
 
-## Support multiple Explore modes
+## Selecting preferred visualisation
 
-Explore lets you query any data source, regardless of whether it returns metrics or logs. You can change which type of query you want to make, by setting the _Explore mode_.
+Explore should be default select a reasonable visualisation for your data so users do not have to tweak and play with the visualisations and just focus on querying. This usually works fairly well and Explore can figure out whether the returned data is time series data or logs or something else.
 
-The query modes that the plugin supports are defined in the [plugin.json]({{< relref "metadata.md" >}}) file.
-
-The query mode is available on the `props` object for both the query editor and the start page. For example, here's how you can change the query editor based on the currently selected mode:
+Occasionally this may not work or you really want to show some data in a specific visualisation. In that cases you can add a hint to your returned data frame, setting the preferredVisualisationType meta attribute.
```

Comment:
```suggestion
Occasionally this may not work or you may want to show some data in a specific visualization. In that case, you can add a hint to your returned data frame, setting the `preferredVisualisationType` meta attribute.
```

Reply:
I recommend replacing "may" with "might." 

Per the [Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may): "When ability is what you need to express, it's OK to use can to describe actions or tasks that the reader or program is able to do. Use _might_ to express possibility. Don't use _may_, which might be interpreted as providing permission."

Reply:
hmm in the case of `or you may want to' -> `or you might want to` does that make sense? Seems like a strange way to say it.

Reply:
For consistency, use _might_ for probability and _may_ for permission. +1 for @oddlittlebird’s comment.

The reason why this might **sound** odd, is that it is largely what is used in spoken language versus written language.